### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by laishchuk**

### CODE REVIEW

File: src/makeCalculator.js [hunk 1]
```diff
@@ -4,7 +4,37 @@
...
+      return b - a;
+    },
+
+    multiply(b, a) {
+      return b * a;
+    },
+
+    divide(b, a) {
+      return b / a;
+    },
```
Comment (volodymyr-soltys97) [inline src/makeCalculator.js:30]: You need to add checking by 0, because you cannot divide by 0

File: src/makeCalculator.js [hunk 2]
```diff
@@ -4,7 +4,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    operate(callback, a) {
+      this.result = callback(this.result, a);
```
Comment (volodymyr-soltys97) [inline src/makeCalculator.js:11]:
```suggestion
    operate(callback, value) {
      callback.call(result, value);
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's make your code better

File: src/makeCalculator.js (hunk 1 above)
Comment (laishchuk) [reply src/makeCalculator.js:30]: divide(b, a) {
      if (a !== 0) {
        return b / a;
      }
    },

Correct checking?

File: src/makeCalculator.js (hunk 2 above)
Comment (laishchuk) [reply src/makeCalculator.js:11]: After this change my code working wrong. Can i keep my option?

**Pull request closed without merging**